src/MLJDecisionTreeInterface.jl

```diff
@@ -379,6 +456,8 @@ Train the machine using `fit!(mach, rows=...)`.
 
 - `display_depth=5`:       max depth to show when displaying the tree
 
+- `feature_importance`:    method to use for computing feature importances
```

Comment:
Sorry, I think we should be a little more descriptive here.
```suggestion
- `feature_importance = :impurity`:   method to use for computing feature importances. One of `(:impurity, :split)`
```